bats/create-tables.bats

```diff
@@ -202,18 +199,23 @@ teardown() {
 @test "import an .xlsx file that is not a valid excel spreadsheet" {
     run dolt table import -c --pk=id test `batshelper bad.xlsx`
     [ "$status" -eq 1 ]
-    skip "errors with 'cause: zip: not a valid zip file'. should say not a valid xlsx file"
     [[ "$output" =~ "not a valid xlsx file" ]] || false
     run dolt ls
     [ "$status" -eq 0 ]
     [[ ! "$output" =~ "test" ]] || false
 }
 
 @test "import a table with non UTF-8 characters in it" {
+    # run dolt table import -c --pk=pk test `batshelper bad-characters.csv`
+    # skip "Dolt allows you to create tables with non-UTF-8 characters right now"
+    # [ "$status" -eq 1 ]
+    # [[ "$output" =~ "unsupported characters" ]] || false
+
     run dolt table import -c --pk=pk test `batshelper bad-characters.csv`
-    skip "Dolt allows you to create tables with non-UTF-8 characters right now"
-    [ "$status" -eq 1 ]
-    [[ "$output" =~ "unsupported characters" ]] || false
+    [ "$status" -eq 0 ]
+    dolt sql -q 'select * from test'
+    dolt sql -r csv -q 'select * from test' > compare.csv
+    diff compare.csv `batshelper bad-characters.csv`
```

Comment:
I changed this test to assert that we store/retrieve non-UTF8 characters correctly. 

We can validate that strings are UTF8 and error, but it's even easier to just support everything